### PR TITLE
Strange error messages stored in question text field #4421

### LINKED ISSUE #4421: Strange error messages stored in question text field
I found this in the questionText  field

```
{ "numOfMcqChoices": 3, "mcqChoices": [ "Yes", "No", "(N/A) Not Scribe" ], "otherEnabled": false, "generateOptionsFor": "NONE", "MIN_NUM_OF_MCQ_CHOICES": 2, "ERROR_NOT_ENOUGH_MCQ_CHOICES": "Too little choices for Multiple-choice (single answer) question. Minimum number of options is: ", "ERROR_INVALID_OPTION": " is not a valid option for the Multiple-choice (single answer) question.", "questionType": "MCQ", "questionText": "The “scribe” listened to and captured all team members input." }
```

Why do we have error messages in question text?

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #4421

### CODE REVIEW

**Comment by damithc** [comment]
Remember to check for compatibility with legacy data.

**Comment by LowWeiLin** [comment]
The stored values will no longer be used, but they will remain in the db until the question is updated.
I could write a script to update the questions, but the cost may be more than simply leaving the extra data in the db.

**Comment by damithc** [comment]
It's fine to leave them in the database.

File: src/main/java/teammates/common/util/Const.java [hunk 1]
```diff
@@ -1183,6 +1185,50 @@ private void _______other_constants________________________(){}
             NOT_SURE,
             NOT_SUBMITTED);
     
+    // Mcq
+    public static final int FEEDBACK_QUESTION_MCQ_MIN_NUM_OF_CHOICES = 2;
```
Comment (kanghj) [inline src/main/java/teammates/common/util/Const.java]: Group these constants under a new class, if possible

**Pull request merged**